pandas/core/internals/blocks.py

```diff
@@ -1680,14 +1688,16 @@ def putmask(
 
         Parameters
         ----------
-        mask  : the condition to respect
+        mask : the condition to respect
         new : a ndarray/object
-        align : boolean, perform alignment on other/cond, default is True
-        inplace : perform inplace modification, default is False
+        align : bool, default True.
+            Perform alignment on other/cond.
+        inplace : bool, default False.
+            Perform inplace modification.
```

Comment:
missing `axis` and `transpose` parameters and descriptions.

personally, i'd be happy following the google style here...

"A method that overrides a method from a base class may have a simple docstring sending the reader to its overridden method’s docstring, such as """See base class.""". The rationale is that there is no need to repeat in many places documentation that is already present in the base method’s docstring. However, if the overriding method’s behavior is substantially different from the overridden method, or details need to be provided (e.g., documenting additional side effects), a docstring with at least those differences is required on the overriding method."

This docstring appears to be duplicate of Block.putmask with a minor inconsistency. i.e. `Return the resulting blocks`. vs `return the resulting block`

Reply:
adding the missing axis and transpose params ill do now, but for the rest (here and in other comments), can we _not_ make this the PR where we start implementing the discussed standards?  This is preliminary to more-important follow-ups.

Reply:
> can we _not_ make this the PR where we start implementing the discussed standards

IIUC these are our standards already in existence for changes made in this PR. 

As a responsible reviewer, just pointing out where the additions/changes in this PR don't meet those standards.

If you feel that this is an unnecessary distraction or disruptive to workflow, then that could be a valid argument in support for a more lightweight standard  for internal docstrings.

Reply:
Totally reasonable, I rescind my request.

Used the see-parent-class-docstring pattern.

Reply:
> If you feel that this is an unnecessary distraction or disruptive to workflow, then that could be a valid argument in support for a more lightweight standard for internal docstrings.

You could also say that this more lightweight standard for internal docstrings already exist: we typically don't review those in such detail on formatting issues. 
(and to be clear: I think it is good to have this discussion (but in the other issue), as it is certainly not clear or some kind of unspoken rule)

Reply:
> You could also say that this more lightweight standard for internal docstrings already exist: we typically don't review those in such detail on formatting issues.

I'm hoping that we can add validation of the internal docstrings to the CI and the CI won't be so lenient.

> I think it is good to have this discussion (but in the other issue)

yes. sorry @jbrockmendel